packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -2,25 +2,28 @@ $usa-banner-background-color: #f0f0f0;
 $usa-banner-text-color: #1b1b1b;
 $usa-banner-close-image-background-color: #e6e6e6;
 $usa-banner-header-action-color: #005ea2;
+$usa-banner-font-family: 'Source Sans Pro Web', 'Helvetica Neue', Helvetica, Roboto, Arial,
+  sans-serif;
 
 .ds-c-usa-banner {
   background-color: $usa-banner-background-color;
   color: $usa-banner-text-color;
-  font-family: 'Source Sans Pro Web', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;
-  font-size: 1.06471rem;
-  line-height: 1.52155;
+  font-family: $usa-banner-font-family;
+  font-size: 1rem;
   padding: 0;
 }
 
 .ds-c-usa-banner__header {
   align-items: flex-start;
   display: flex;
   flex-direction: row;
-  font-size: 0.79853rem;
+  font-size: 0.8rem;
   font-weight: 400;
-  line-height: 1.12707;
+  line-height: inherit;
```

Comment:
Is this line necessary? Does the style change if you just delete it?

Reply:
Needed this, otherwise style will change.